package-res/resources/web/pentaho/visual/base/View.js

```diff
@@ -94,37 +91,97 @@ define([
     /**
      * Orchestrates the rendering of the visualization.
      *
-     * This method executes [_render]{@link pentaho.visual.base.View#_render}
+     * Executes the will/did/rejected event loop associated with an update in a visualization.
+     *
```

Comment:
IMO, we should emphasize in the doclet that the DOM element is created on the first execution of `#update` and that the container can only get a reference to the DOM element by listening to the `did:create` event.